File: net/src/Sdl.Web.GraphQL/Response/GraphQLResponse.cs
using System.Collections.Generic;
using Sdl.Web.HttpClient;

namespace Sdl.Web.GraphQLClient.Response
{
    /// <summary>
    /// Represents the GraphQL respinse
    /// </summary>
    public class GraphQLResponse : IGraphQLResponse
    {
        /// <summary>
        /// GraphQL Data
        /// </summary>
        public dynamic Data { get; set; }

        /// <summary>
        /// GraphQL Errors
        /// </summary>
        public List<GraphQLError> Errors { get; set; }

        /// <summary>
        /// Response headers
        /// </summary>
        public HttpHeaders Headers { get; set; }
    }
}

File: java/GraphLClient/src/main/java/com/sdl/web/Response/GraphQLResponse.java
package com.sdl.web.Response;

import java.util.List;

public class GraphQLResponse implements IGraphQLResponse{
    private Object Data;
    private List<GraphQLError> Errors;

    @Override
    public void setData(Object data) {
        Data = data;
    }

    @Override
    public void setErrors(List<GraphQLError> errors) {
        Errors = errors;
    }

    @Override
    public Object getData() {
        return Data;
    }

    @Override
    public List<GraphQLError> getErrors() {
        return Errors;
    }
}
